<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="7031" started-at="2021-11-11T18:04:26 IST" finished-at="2021-11-11T18:04:33 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="7031" started-at="2021-11-11T18:04:26 IST" finished-at="2021-11-11T18:04:33 IST">
      <class name="thirdScenario.PepperfryWebsite">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:thirdScenario.PepperfryWebsite@6a1aab78]" name="beforeTest" is-config="true" duration-ms="5707" started-at="2021-11-11T18:04:26 IST" finished-at="2021-11-11T18:04:32 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="productlist(java.lang.String)[pri:0, instance:thirdScenario.PepperfryWebsite@6a1aab78]" name="productlist" duration-ms="0" started-at="2021-11-11T18:04:32 IST" data-provider="dp" finished-at="2021-11-11T18:04:32 IST">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Data Provider public java.lang.Object[][] thirdScenario.PepperfryWebsite.dp() returned a null value]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.Object[][] thirdScenario.PepperfryWebsite.dp() returned a null value
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:148)
at org.testng.internal.Parameters.handleParameters(Parameters.java:798)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- productlist -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:thirdScenario.PepperfryWebsite@6a1aab78]" name="afterTest" is-config="true" duration-ms="1115" started-at="2021-11-11T18:04:32 IST" finished-at="2021-11-11T18:04:33 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- thirdScenario.PepperfryWebsite -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
